configs/setups/awicm3/awicm3.yaml

```diff
@@ -30,7 +34,7 @@ general:
             add_include_models:
             - xios
 
-        version: "1.0-deck"
+        versions: "1.0-deck"
```

Comment:
Does it work really with `version` instead of `versions`?

Reply:
no that looks like a typo. strange that is caused no error. but im not using 1.0-deck here. I'll fix it